### PR TITLE
feat: support prisma 2

### PR DESCRIPTION
**Pull request by jasonkuhrt**
BREAKING CHANGE:

- Requires Prisma 2.0.0 or higher

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "jest": "25.2.4",
     "jest-watch-typeahead": "0.4.2",
     "node-fetch": "2.6.0",
+    "p-queue": "^6.4.0",
```
Comment (jasonkuhrt) [inline package.json]: This can be removed in latest alpha https://github.com/prisma/prisma/issues/2692

**Comment by jasonkuhrt** [comment]
Some odd things

test suite not exiting correctly

image

funky things with field-resolutions test 

image

File: package.json (hunk 1 above)
Comment (Weakky) [reply package.json]: Should be removable now that 2.0.0 has been released, right ?

Comment (jasonkuhrt) [reply package.json]: I'll revise this PR to be about 2.0 and yeah let's see.

Comment (jasonkuhrt) [reply package.json]: Ok works.

**Comment by jasonkuhrt** [comment]
https://github.com/prisma/prisma/issues/2703

**Review by Weakky — APPROVED** [review]
Looks all good to me 💯

**Pull request merged**